test/test.js

```diff
@@ -2,8 +2,8 @@
 const test = require('ava');
 
 // Utilities
-const UIDCHARS = require('../lib/chars');
-const uid = require('../lib');
+const UIDCHARS = require('../lib/cjs/chars');
+const uid = require('../lib/cjs');
```

Comment:
Perhaps the tests should be updated to include another case for esm to make sure that works?

Reply:
Hmm I tried this - but getting errors from `ava` saying `SyntaxError: Unexpected token 'export'`.  Looking into that it looks like we would have to transpile our ESM code back to CJS in order for ava to work 😂 

I could add a script that adds --experimental-global-webcrypto to the node options for ava - forcing the test to use the "browser" code

Reply:
@agadzik You might need to add `lib/esm/package.json` with `type: module`. Or you could try changing `lib/esm/index.js` to `lib/esm/index.mjs`.

But now that I think about it, maybe `ava` doesn't support ESM at all 🤷‍♂️

Reply:
> lib/esm/package.json with type: module

This was the golden ticket, thank you!